tensorflow/lite/examples/label_image/label_image.cc

```diff
@@ -13,40 +13,87 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
-#include <fcntl.h>      // NOLINT(build/include_order)
-#include <getopt.h>     // NOLINT(build/include_order)
-#include <sys/time.h>   // NOLINT(build/include_order)
-#include <sys/types.h>  // NOLINT(build/include_order)
-#include <sys/uio.h>    // NOLINT(build/include_order)
-#include <unistd.h>     // NOLINT(build/include_order)
-
 #include <cstdarg>
 #include <cstdio>
 #include <cstdlib>
 #include <fstream>
 #include <iomanip>
 #include <iostream>
+#include <map>
 #include <memory>
 #include <sstream>
 #include <string>
 #include <unordered_set>
 #include <vector>
 
-#include "tensorflow/lite/examples/label_image/bitmap_helpers.h"
-#include "tensorflow/lite/examples/label_image/get_top_n.h"
+#include <fcntl.h>      // NOLINT(build/include_order)
```

Comment:
Why shift these down?

Reply:
separate `#include <>`s and `#include ""`s

Reply:
I see, though the convention followed in the rest of the codebase is:
1. `<xyz.h>` includes
2.  `<>` includes
3. `""` includes
...with a newline between each block.
Could you restructure it that way?

Reply:
done